### PR TITLE
Better data for ExampleCostInsightsClient

### PR DESCRIPTION
**Pull request by timbonicus**
Enhances the ExampleCostInsightsClient to return random cost data appropriate for the selected intervals, rather than incomplete hard-coded values.

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [x] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: packages/app/package.json [hunk 1]
```diff
@@ -36,13 +36,15 @@
     "@roadiehq/backstage-plugin-github-pull-requests": "0.3.0",
     "@roadiehq/backstage-plugin-travis-ci": "^0.2.3",
     "history": "^5.0.0",
+    "moment": "^2.27.0",
```
Comment (nikek) [inline packages/app/package.json:39]: Maybe we could use the much smaller [dayjs](https://github.com/iamkun/dayjs) instead :)

File: yarn.lock [hunk 2]
```diff
@@ -19842,6 +19847,11 @@ regjsparser@^0.6.4:
   dependencies:
     jsesc "~0.5.0"
 
+regression@^2.0.1:
+  version "2.0.1"
+  resolved "https://artifactory.spotify.net/artifactory/api/npm/virtual-npm/regression/-/regression-2.0.1.tgz#8d29c3e8224a10850c35e337e85a8b2fac3b0c87"
```
Comment (marcuseide) [inline yarn.lock:19852]: Ah, looks like this made it's way here. I think you want to make sure the npm registry is correctly set when adding packages. `YARN_REGISTRY=https://registry.npmjs.org yarn add ...`

**Review by marcuseide — DISMISSED** [review]

File: packages/app/package.json (hunk 1 above)
Comment (timbonicus) [reply packages/app/package.json:39]: We have a task to switch this out in our next iteration 👍

File: yarn.lock (hunk 2 above)
Comment (timbonicus) [reply yarn.lock]: Ah, good catch. There's an `.npmrc` with the standard registry in root and our plugin. Did I run yarn install from elsewhere? A mystery for the ages..

**Review by ryanvazquez — APPROVED** [review]
🔥

File: packages/app/package.json [hunk 3]
```diff
@@ -55,6 +57,7 @@
     "@types/jquery": "^3.3.34",
```
Comment (ryanvazquez) [inline packages/app/package.json:57]: jQuery ?? #whatyearisit

Comment (freben) [reply packages/app/package.json:57]: I think you need a Unix Beard to use that

**Comment by timbonicus** [comment]
@spotify/backstage-core Can you have another look? I'm a bit confused by the codecov check failures.

**Comment by Rugvip** [comment]
Also confused, @OrkoHunter, know what's going on?

**Comment by Rugvip** [comment]
@timbonicus Fix the conflicts and we can merge it anyway 😅

**Review by Rugvip — APPROVED** [review]

File: packages/app/package.json (hunk 1 above)
Comment (timbonicus) [reply packages/app/package.json]: I switched to dayjs for this PR in `app`, will look at updating cost-insights with the same soon.

**Pull request merged**